doc/Language/functions.pod

```diff
@@ -724,9 +724,9 @@ currently (2015.02) only implements them for subroutine parameters.
 
 =head1 C<sub MAIN>
 
-The sub with the special name C<MAIN> is executed after all relevant phaser
-and it's signature is the means by which command line arguments can be parsed.
-Multi methods are supported and a usage method is automatically generated and
+The sub with the special name C<MAIN> is executed after all relevant phases
```

Comment:
FWIW, the `phaser` is the correct word (just has to be pluralized to `phasers`). It's talking about phasers like BEGIN, INT, etc